app/models/ingest_job.rb

```diff
@@ -484,22 +484,20 @@ def get_annotation_message(annotation_source:, cell_annotation: nil)
     case annotation_source.class
     when CellMetadatum
       message = "#{annotation_source.name}: #{annotation_source.annotation_type}"
-      if annotation_source.can_visualize? || annotation_source.values.size == 1
+      if annotation_source.values.size < CellMetadatum::GROUP_VIZ_THRESHOLD.max || annotation_source.annotation_type == 'numeric'
         values = annotation_source.values.any? ? ' (' + annotation_source.values.join(', ') + ')' : ''
       else
         values = ' (List too large for email)'
```

Comment:
Non-blocking refinement to consider:

It'd be nice if this message had slightly more detail, e.g. how large the list was (`annotation_source.values.size`), and how large is too large for email (`CellMetadatum::GROUP_VIZ_THRESHOLD.max`).

Reply:
That's a nice refinement - I'll incorporate that.

Reply:
3975cd3